tests/scf-cholesky-basis/input.dat

```diff
@@ -19,9 +19,9 @@ e_canonical   = -230.64152543331906
 tol_4 = 3
 tol_5 = 4
 tol_6 = 5
-tol_7 = 8
-tol_8 = 8
-tol_9 = 8
+tol_7 = 7
+tol_8 = 7
+tol_9 = 7
```

Comment:
oh, wasn't the plan to try `2.e-8`?

Reply:
The first time you said anything about that was _after_ my last commit.

Reply:
sorry, you were ahead of me :-) I do think it'd be better to stay conservative, but we can patch it up in another PR and save cycles. Whichever you prefer.

Reply:
I agree that conservative is better; I just was used to us sticking to `1e-X` checks.

You patching it up in upcoming docs PR to save cycles sounds good, if you're willing.